lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/lib/main.js

```diff
@@ -18,20 +18,64 @@
 
 'use strict';
 
+// MODULES //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+
+
 // MAIN //
 
 /**
 * Evaluates a normalized Hermite polynomial.
 *
-* @param {<TODO: type>} TODO - TODO: param desc
-* @returns {<TODO: type>} TODO: desc
+* @param {integer} n - degree of polynomial
+* @param {number} x - evaluation point
+* @returns {number} function value
+*
+* @example
+* var v = normhermitepoly( 1, 0.5 );
+* // returns 0.5
+*
+* @example
+* var v = normhermitepoly( -1, 0.5 );
+* // returns 0.0
 *
 * @example
-* var v = normhermitepoly( 1, 1.0 );
-* // returns TODO
+* var v = normhermitepoly( 0, 0.5 );
+* // returns 1.0
+*
+* @example
+* var v = normhermitepoly( 2, 0.5 );
+* // returns -0.75
 */
-function normhermitepoly() {
-	// TODO: implementation
+function normhermitepoly( n, x ) {
+	var y1;
+	var y2;
+	var y3;
+	var i;
+
+	if ( n === 0 ) {
+		// The x gets completely cancelled from the expression
+		return 1.0;
+	}
+	if ( isnan( x ) ) {
```

Comment:
You'll most likely want to perform `NaN` checks before returning sensible results. Why? Because if a user provides `NaN` for either `n` or `x`, likely that some upstream calculation went wrong. Returning a sensible result would mean that `NaNs` won't propagate (poison future calculations), making debugging more difficult, etc.